### ISSUE TITLE
improve: `liara logs` should consider liara.json file

### ISSUE THREAD

**Issue by erfanium** [liara-cloud/cli#152]
I already have an `liara.json` file in the root of my project and i expect liara to use the `appName` from this file when I use `liara logs`.

https://github.com/liara-cloud/cli/blob/5be871541d670ab19a0188e6073db4bb4ee3c8d9/src/commands/app/logs.ts#L59

should be changed to something like:
```ts
const project = flags.app || (await readFromLiaraFile()) || (await this.promptProject()); 
```

**Comment by mhemrg** [comment]
Hi @erfanium,
Thanks for your feedback!

This has been addressed in #153, so you can try it using our beta version for now: `npm i -g @liara/cli@beta`.

**Comment by mhemrg** [comment]
You can now install the latest version: `npm i -g @liara/cli`
